src/frontend/components/PublicLiveDashboard/index.tsx

```diff
@@ -22,14 +25,30 @@ export const PublicLiveDashboard = ({
 }: PublicLiveDashboardProps) => {
   const intl = useIntl();
   const [isReadyLive, setIsReadyLive] = useState(false);
-
-  useAsyncEffect(async () => {
-    if (!isReadyLive) {
-      await initWebinarContext(live, intl.locale);
-      initVideoWebsocket(live);
-      setIsReadyLive(true);
-    }
-  }, [isReadyLive, live]);
+  const anonymousId = useMemo(() => getOrInitAnonymousId(), []);
+  const queryClient = useQueryClient();
+  const setLiveSession = useLiveSession((state) => state.setLiveSession);
+  useLiveSessions(
+    { anonymous_id: anonymousId },
+    {
+      onSuccess: async (data) => {
```

Comment:
Shouldn't we define the onError as well ?
React query retries by default 3 times but the websocket might not be initialized and the user will be stuck on a loader if all 3 attempts fail.